sap/cf_logging/falcon_logging/request_reader.py

```diff
@@ -0,0 +1,39 @@
+""" Falcon request reader """
+
+import base64
+import binascii
+import logging
+
+from sap.cf_logging import defaults
+from sap.cf_logging.core.request_reader import RequestReader
+
+
+class FalconRequestReader(RequestReader):
+    def get_remote_user(self, request):
+        remote_user = defaults.UNKNOWN
+        http_auth = request.get_header('Authorization')
```

Comment:
I would prefer not to parse auth headers into the logging lib, rather to support the most common authentication frameworks and the way they provide user info. I would suggest to read the user from the request context as expected to be set by [falcon-auth](https://github.com/loanzen/falcon-auth) authentication layer.